### PR TITLE
fix: cannot create `PythonProject` with `.projenrc.js`

### LINKED ISSUE #1807: Python-based projects can no longer use existing .projenrc.js files
I've got a number of projects that are PythonProject based, and now with the latest updates an `npx projen` no longer runs because it's trying to find a .projenrc.py file. I see options to turn that off, as well as turning on a .json option, but I need to still use the existing .projenrc.js files and don't see the option to do it.

To make this work, I've had to go hacking at the default task:

`this.tasks.tryFind('default')?.exec('node .projenrc.js');`

which isn't something I feel I should have to do to maintain backwards compatability.

### PR DESCRIPTION
**Pull request by Chriscbr**
Fixes #1807

To manage a PythonProject with a JavaScript projenrc file, the project will need to have these options configured:
```js
projenrcJs: true,
```

Or pass the option in the command line when creating the project, e.g. `npx projen new python --projenrc-js`.

---
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/python/python-project.ts [hunk 1]
```diff
@@ -134,7 +141,23 @@ export interface PythonProjectOptions
...
-  readonly projenrcPythonOptions?: ProjenrcOptions;
+  readonly projenrcPythonOptions?: ProjenrcPythonOptions;
+
+  /**
+   * Use projenrc in javascript.
+   *
+   * This will install `projen` as a JavaScript dependency and add a `synth`
+   * task which will run `.projenrc.js`.
+   *
+   * @default true
```
Comment (kaizencc) [inline src/python/python-project.ts:152]: Color me confused... how can both `projenrcJs` and `projenrcPython` default to true?

Comment (kaizencc) [reply src/python/python-project.ts:152]: Wait, this default looks like it should be false from the code.

**Review by kaizencc — CHANGES_REQUESTED** [review]
It seems like if a user wants to use the JavaScript dependency, then must explicitly set `projenrcPython: false`. Seems like a poor user experience, or am I missing something? I feel like setting `projenrcJs: true` (assuming the default is actually false, not true) should be enough signal to assume the user does not want the Python dependency projenrc.

File: src/python/python-project.ts (hunk 1 above)
Comment (Chriscbr) [reply src/python/python-project.ts]: Whoops! good catch

**Comment by Chriscbr** [comment]
Good callouts, I agree that disabling `projenrcPython` in order to use the other options is unnecessary 👍

**Review by kaizencc — APPROVED** [review]

**Pull request merged**